src/query/storages/fuse/fuse/src/fuse_table.rs

```diff
@@ -498,13 +498,17 @@ impl Table for FuseTable {
     }
 
     #[tracing::instrument(level = "debug", name = "fuse_table_navigate_to", skip_all)]
-    async fn navigate_to(&self, point: &NavigationPoint) -> Result<Arc<dyn Table>> {
+    async fn navigate_to(
+        &self,
+        point: &NavigationPoint,
+        read_only: bool,
```

Comment:
The new para `read_only` is confusing:
If `read_only = true` what will do, or  `read_only=false` what will do?

Do we consider a more meaningful `enum` here?

Reply:
if read_only is true, the table instance returned will be read only, i.e. data mutations, like insert / delete, are not allowed(runtime exception).

yes we can replace the bool with an enumeration, such as:

~~~
enum ReadOnly {
  True,
  False,
}
~~~

not sure if it is worth to do it. maybe we can upgrade the type to enum when there are more variants?

Reply:
the `read_only` flag of fuse table instance might be over-cautioned, it seems to be enough to let the execution layer to prevention historical data from being mutated. thus, the flag has been removed.

@BohuTANG  PTAL